cmd/cl-adm/cmd/deploy/deploy_peer.go

```diff
@@ -80,13 +82,18 @@ func NewCmdDeployPeer() *cobra.Command {
 func (o *PeerOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&o.Name, "name", "", "Peer name.")
 	fs.StringVar(&o.CertDir, "cert-dir", ".", "The directory where the certificates for the fabric and peer are located.")
-	fs.BoolVar(&o.StartInstance, "start", false, "If true, deploy a ClusterLink instance that will create ClusterLink components")
-	fs.StringVar(&o.Namespace, "start-namespace", app.SystemNamespace,
-		"Namespace where the ClusterLink components are deployed if --start is set.")
-	fs.StringVar(&o.Ingress, "start-ingress", string(apis.IngressTypeLoadBalancer),
-		" By default, it is set to 443 for LoadBalancer and a random port (3000 to 32767) for NodePort, if --start is set.")
+	fs.StringVar(&o.Namespace, "namespace", app.SystemNamespace,
```

Comment:
do we need a flag for the operator namespace? It seems that this is for the CL components so the operator is fixed?
Also, shouldn't this have an associated `--autostart` message?

Reply:
This is only for automation, if someone wants to create the operator in a different namespace (like Openshift) it will not use this CLI.
The CLI also creates the secrets in the namespace so it is always needed.